src/OpenTelemetry.Exporter.Stackdriver/Implementation/ActivityExtensions.cs

```diff
@@ -15,15 +15,15 @@
 // </copyright>
 
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using Google.Cloud.Trace.V2;
 using Google.Protobuf.WellKnownTypes;
 using OpenTelemetry.Trace;
-using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.Stackdriver.Implementation
 {
-    internal static class SpanExtensions
```

Comment:
Looks like some name refactoring has been included here - please move to separate PR :+1:

Reply:
I renamed because when we updated, the project didn't compile because we don't expose SpanExporter anymore.

Reply:
Shouldn't the build be failing on master if `SpanData` is no longer exposed?

Reply:
well, in master, we was using an alpha version, which still had that version.

Reply:
I'm not really following - the changes to update the package feed to use the new beta should be isolated from the refactoring. The current master branch, currently pointing at myget, should have failed because if `SpanData` was missing.

Unless the version that was pointed to in the myget feed was behind master? In that case, I'd still prefer to see separate PRs to do the refactoring and update the package feed. Separate PRs help to reduce PR complexity and make smaller changes that can be reverted if required.

Reply:
@MikeGoldsmith , i updated not only the feed, but the version as well. We were using 0.2.0-alpha.404 and now, with this pr, we are using 0.4.0-beta.2. With this change, SpanData doesn't exist, so, I had to update to Activity style. What i can do is:
1. update to a beta inside myget, To the latest one (that will include the refactor to activity style)
2. update the feed from myget to nuget